src/lib/y2storage/lvm_vg.rb

```diff
@@ -216,6 +216,68 @@ def potential_orphans
       lvm_pvs
     end
 
+    # Size of the physical volume that limits the maximum size of a new striped logical volume
+    #
+    # The maximum size of a striped logical volume is limited by the n-th biggest physical volume. For
+    # example, let's say we have a volume group with 3 physical volumes: pv1 (2 GiB), pv2 (50 GiB) and
+    # pv3 (1 GiB). For a striped volume with 2 stripes, the second biggest physical volume (pv1)
+    # restricts its maximum size. In this case, the maximum size would be:
+    # 2 (stripes) * 2 GiB (pv1 size) = 4 GiB.
+    # If the number of stripes is 3, then the maximum size is limited by the third biggest physical
+    # volume (pv3), so the maximum size would be: 3 (stripes) * 1 GiB (pv3 size) = 3 GiB.
+    #
+    # @param stripes [Integer] number of stripes. It must be bigger than 1
+    #
+    # @raise [RuntimeError] when the given number of stripes is incorrect (i.e., less than 1)
+    # @return [DiskSize, nil] nil if no enough physical volumes for the given number of stripes
+    def pv_size_for_striped_lv(stripes)
+      raise "stripes must be bigger than 1" unless stripes > 1
+
+      return nil if stripes > lvm_pvs.size
+
+      lvm_pvs.map(&:blk_device).map(&:size).sort[-stripes]
+    end
+
+    # Maximum size for a new striped logical volume
+    #
+    # @note This size is not 100% accurate. It is based on the size of the block devices used as physical
+    #   volumes. But LVM probably uses some space to allocate its metadata. libstorage-ng provides some
```

Comment:
It's not only about the LVM metadata. If I'm not mistaken, this method also assumes the stripped LV is the only one having some space in the limiting PV. In other words, this does not have into account the space already used by other planned or pe-existing LVs.

And about the LVM metadata, we already have such content elsewhere. Let me check.

Reply:
> And about the LVM metadata, we already have some logic somewhere to deal with that. Let me check.

It's in `Y2Storage::Planned::LvmVg#useful_pv_space` and similar methods.

Reply:
I have just realized that *libstorage-ng* already provides a method to get the usable size, see https://github.com/openSUSE/libstorage-ng/blob/master/storage/Devices/LvmPv.h#L85.